PSSwagger/Definitions.psm1

```diff
@@ -582,6 +582,7 @@ function Expand-SwaggerDefinition
                                                                     else
                                                                     {
                                                                         $FunctionDetails.ParametersTable[$RefParameterName] = $RefFunctionDetails.ParametersTable[$RefParameterName]
+                                                                        $FunctionDetails.ParametersTable[$RefParameterName]['Source'] = $ReferencedDefinitionName
```

Comment:
Since this is a reference assignment, $ReferencedDefinitionName assignment may change the value of earlier assigned parametersTable values.
Consider copying the $RefFunctionDetails.ParametersTable[$RefParameterName] in above assignment.